server/engine/telemetry.go

```diff
@@ -117,6 +118,22 @@ func SetFinalMetrics(db *gorm.DB) {
 
 }
 
+func UpdateLogInMetrics(db *gorm.DB) {
+	loginsCount := "1" // default value for case if no logins were recorded yet
```

Comment:
Why not zero?

Reply:
This function is called only after successful login so this default of 1 means we don't have to do a +1 operation when no previous logins were recorded. If there were previous logins, the +1 is done right there (line 130)